app/src/main/java/org/sagebionetworks/bridge/researchstack/CrfDataProvider.java

```diff
@@ -18,10 +18,6 @@
  */
 
 public class CrfDataProvider extends BridgeDataProvider {
-
-    public static final String CLINIC1 = "clinic1";
```

Comment:
Keep these here, I use them in 
https://github.com/dephillipsmichael/CardiorespiratoryFitness-Android/commit/c392237fbd3d5a2a48af21dbac8f9fd91021a077